example/onelogin_custom_settings/settings.php

```diff
@@ -0,0 +1,138 @@
+        'assertionConsumerService' => array (
+            'url' => $acs_endpoint
+        ),
+        'singleLogoutService' => array (
+            'url' => add_query_arg( [ 'saml_sls' => '' ], wp_login_url() )
+        ),
+        'NameIDFormat' => $opt['NameIDFormat'],
+        'x509cert' => get_option('onelogin_saml_advanced_settings_sp_x509cert'),
+        'privateKey' => get_option('onelogin_saml_advanced_settings_sp_privatekey'),
+        #ITWallet: Adding the attributeConsumingService section in the metadata
+        'attributeConsumingService' => array (
+            'serviceName' => (!empty($opt['sp_entity_id'])? $opt['sp_entity_id'] : 'php-saml'),
+            'requestedAttributes' => array (
+                '0' => array (
+                    'name' => 'spidCode',
+                    'isRequired' => true
+                ),
+                '1' => array (
+                    'name' => 'name',
+                    'isRequired' => true
+                ),
+                '2' => array (
+                    'name' => 'familyName',
+                    'isRequired' => true
+                ),
+                '3' => array (
+                    'name' => 'fiscalNumber',
+                    'isRequired' => true
+                ),
+                '4' => array (
+                    'name' => 'email',
+                    'isRequired' => true
+                )
+            )
+        )
+    ),
+
+    'idp' => array (
+        'entityId' => get_option('onelogin_saml_idp_entityid'),
+        'singleSignOnService' => array (
+            'url' => get_option('onelogin_saml_idp_sso'),
+        ),
+        'singleLogoutService' => array (
+            'url' => get_option('onelogin_saml_idp_slo'),
+        ),
+        'x509cert' => get_option('onelogin_saml_idp_x509cert'),
+    ),
+
+    'security' => array (
+        'signMetadata' => true, #ITWallet: enables metadata signature
+        'nameIdEncrypted' => $opt['nameIdEncrypted'] == 'on'? true: false,
+        'authnRequestsSigned' => $opt['authnRequestsSigned'] == 'on'? true: false,
+        'logoutRequestSigned' => $opt['logoutRequestSigned'] == 'on'? true: false,
+        'logoutResponseSigned' => $opt['logoutResponseSigned'] == 'on'? true: false,
+        'wantMessagesSigned' => $opt['wantMessagesSigned'] == 'on'? true: false,
+        'wantAssertionsSigned' => $opt['wantAssertionsSigned'] == 'on'? true: false,
+        'wantAssertionsEncrypted' => $opt['wantAssertionsEncrypted'] == 'on'? true: false,
+        'wantNameId' => false,
+        'requestedAuthnContext' => $opt['requestedAuthnContext'],
+        'relaxDestinationValidation' => true,
+        'lowercaseUrlencoding' => get_option('
+            onelogin_saml_advanced_idp_lowercase_url_encoding', false),
+        'signatureAlgorithm' => get_option('onelogin_saml_advanced_signaturealgorithm', 'http://www.w3.org/2000/09/xmldsig#rsa-sha1'),
```

Comment:
use sha256 instead of sha1

Reply:
resolved with [9047963c1abd51a8505bc09abb07f86f0f387cc3](https://github.com/italia/eudi-wallet-it-python/pull/75/commits/9047963c1abd51a8505bc09abb07f86f0f387cc3)